### PR TITLE
Fix extension warnings

### PR DESCRIPTION
**Pull request by rcedwards**
### Summary of Changes

Sets the xcode project flag for extension safe API to YES

### Addresses

Warnings when used as a framework on other projects #115

### CODE REVIEW

**Comment by rcedwards** [comment]
@zwaldowski have a second to scan this? thanks

File: Example/BooksTableViewController.swift [hunk 1]
```diff
@@ -51,7 +51,7 @@ class BooksTableViewController: UITableViewController {
 
         navigationItem.rightBarButtonItem = UIBarButtonItem(barButtonSystemItem: .Done,
             target: self,
-            action: "dismiss")
+            action: #selector(BooksTableViewController.dismiss))
```
Comment (zwaldowski) [inline Example/BooksTableViewController.swift]: Should just be `#selector(dismiss)`

File: CoreDataStack.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -1141,6 +1148,7 @@
 		E4C9BC481AEA848600A6BD1B /* Release */ = {
 			isa = XCBuildConfiguration;
 			buildSettings = {
+				APPLICATION_EXTENSION_API_ONLY = NO;
```
Comment (zwaldowski) [inline CoreDataStack.xcodeproj/project.pbxproj]: Under what conditions is it being set to `NO`?

Comment (rcedwards) [inline CoreDataStack.xcodeproj/project.pbxproj]: these are for the testing targets b/c of XCTest

Comment (zwaldowski) [inline CoreDataStack.xcodeproj/project.pbxproj]: Got it

**Pull request merged**